orca-queue/orca-queue.gradle

```diff
@@ -40,5 +40,4 @@ dependencies {
   testImplementation(project(":orca-queue-redis"))
   testImplementation(project(":orca-echo"))
   testImplementation("org.assertj:assertj-core")
-  testImplementation("org.codehaus.groovy:groovy-all:3.0.12")
```

Comment:
Sorry for the thrashing here.  At least for me locally, I need something like
```
  testImplementation("org.codehaus.groovy:groovy")
```
otherwise I see:
```
> Task :orca-queue:compileTestKotlin
e: Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class com.netflix.spinnaker.orca.echo.pipeline.ManualJudgmentStage, unresolved supertypes: groovy.lang.GroovyObject

e: /Users/dbyron/src/spinnaker/orca/orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt: (1860, 38): Cannot access 'groovy.lang.GroovyObject' which is a supertype of 'com.netflix.spinnaker.orca.echo.pipeline.ManualJudgmentStage'. Check your module classpath for missing or conflicting dependencies
e: /Users/dbyron/src/spinnaker/orca/orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt: (1871, 36): Cannot access 'groovy.lang.GroovyObject' which is a supertype of 'com.netflix.spinnaker.orca.echo.pipeline.ManualJudgmentStage'. Check your module classpath for missing or conflicting dependencies
```

Reply:
I removed it at all, I ran the tests, and are passing without problems. I was using the dependency to experiment with ManualJudgmentStage(groovy object)